Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/MicrosoftManagementActivity.py

```diff
@@ -91,12 +89,33 @@ def __init__(self, base_url: str, verify: bool,
                                          proxy=proxy,
                                          refresh_token=self.refresh_token,
                                          ok_codes=(200, 201, 202, 204),
+                                         timeout=self.timeout,
                                          scope='',
                                          auth_code=auth_code,
                                          resource='https://manage.office.com',
                                          token_retrieval_url='https://login.windows.net/common/oauth2/token',
                                          redirect_uri=redirect_uri)
 
+    def http_request(self, method, url_suffix='', full_url=None, headers=None, auth=None, json_data=None,
+                     params=None, data=None, files=None, timeout=None, resp_type='json', ok_codes=None,
+                     return_empty_response=False, retries=0, status_list_to_retry=None,
+                     backoff_factor=5, raise_on_redirect=False, raise_on_status=False,
+                     error_handler=None, empty_valid_codes=None, **kwargs):
+        """
+        Calls the built in http_request, replacing a None timeout with self.timeout
+        """
+        if timeout is None:
+            timeout = self.timeout
+        return self._http_request(method=method, url_suffix=url_suffix, full_url=full_url,
+                                  params=params, data=data, files=files, resp_type=resp_type, ok_codes=ok_codes,
+                                  headers=headers, auth=auth, json_data=json_data,
+                                  return_empty_response=return_empty_response, retries=retries,
+                                  status_list_to_retry=status_list_to_retry,
+                                  backoff_factor=backoff_factor, raise_on_redirect=raise_on_redirect,
+                                  raise_on_status=raise_on_status, error_handler=error_handler,
+                                  empty_valid_codes=empty_valid_codes,
+                                  timeout=timeout, **kwargs)
+
```

Comment:
Does the integration really use all of these args?
No need to reiterate args that have `default values` in `_http_requeest` and are not used by the integration.